scripts/e2etests/VSUtils.ps1

```diff
@@ -207,6 +207,33 @@ function Update-Configuration(
     Start-Process -FilePath $vsFilePath -ArgumentList '/updateConfiguration' -Wait
 }
 
+function ResumeVSInstall {
+    param(
+        [Parameter(Mandatory = $true)]
+        [int]$ProcessExitTimeoutInSeconds
+    )
+
+    $ProgramFilesPath = ${env:ProgramFiles}
+    if (Test-Path ${env:ProgramFiles(x86)}) {
+        $ProgramFilesPath = ${env:ProgramFiles(x86)}
+    }
+    $VSInstallerPath = "$ProgramFilesPath\Microsoft Visual Studio\Installer\vs_installer.exe"
+    $VSFolderPath = GetVSFolderPath 16.0
+
+    Write-Host 'Resuming any incomplete install'
+    $args = "resume --installPath ""$VSFolderPath"" -q"
+    Write-Host """$VSInstallerPath"" $args"
```

Comment:
Feels like you have extra `"` here.

Reply:
no, the first and last `"` defines the string, and in Powershell to put a single double-quote character in a string, you need to write two double quote characters, just like in c# verbatim string literals.

So, if  `$VSInstallerPath = "prog.exe"` and `$args = "-1 -2"`, then `"prog.exe" -1 -2` will be written out.